README.md

```diff
@@ -573,7 +573,7 @@ Configuration also supports fallback values:
 foo: ${env:MY_ENV_VAR, 'default-value'}
 ```
 
-Configuration can be written in `toml`, `yml`, `json`, or `json5`.
+Configuration can be written in `toml`, `yml` or `json`.
```

Comment:
The [`json5 lib`](https://www.npmjs.com/package/json5) looks like an easy way to keep json5 support. Thoughts?

Reply:
We just discussed with @erquhart on Slack to not keep for the moment until more feedback is gathered. There are a few JSON alternatives and some developers have strong opinions about which one is the best, so more feedback would be beneficial.